sktime/utils/validation/series.py

```diff
@@ -18,7 +18,13 @@
 
 # We currently support the following types for input data and time index types.
 VALID_DATA_TYPES = (pd.DataFrame, pd.Series, np.ndarray)
-VALID_INDEX_TYPES = (pd.Int64Index, pd.RangeIndex, pd.PeriodIndex, pd.DatetimeIndex)
+VALID_INDEX_TYPES = (
+    pd.Int64Index,
+    pd.RangeIndex,
+    pd.PeriodIndex,
+    pd.DatetimeIndex,
+    pd.TimedeltaIndex,
```

Comment:
why do we need to add this here? Should the series index not always be absolute?
Only in the forecasting horizon we want to allow relative ones.

Is there something I`m missing?

Reply:
I added `pd.TimedeltaIndex` to `RELATIVE_TYPES` in `_fh.py`. Otherwise, I hit this exception with my test cases:
```
raise TypeError(f"{type(values)} is not a supported fh index type")
```
If I do not add the same index type to `VALID_INDEX_TYPES` in `series.py`, then I hit this roadblock:
```
assert set(RELATIVE_TYPES).issubset(VALID_INDEX_TYPES)
```
Any suggestions to circumvent this?

Reply:
no, this makes sense. This includes both absolute and relative, my mistake.